src/style.css

```
@@ -0,0 +1,160 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(./fonts/Avenir-Book.ttf);
+
+}
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  max-width: 100%;
+  font-weight: 300px;
+}
+
+.wrapper {
+  max-width: 100%;
+  max-height: 68px;
+}
+/* Form_big */
+.radius {
+  max-width: 100%;
+  max-height: 68px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  margin-top: 20px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.radius:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.radius:has(.input_big:focus) {
+  background: linear-gradient(#fff, #f6f6f7);
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 4px;
+}
+
+.radius:has(.input_small:focus) {
+  background: linear-gradient(#fff, #f6f6f7);
+}
+
+.form_big {
+  display: flex;
+  max-width: 100%;
+  max-height: 68px;
+}
+
+.input_big {
+  display: flex;
+  padding: 0;
+  width: 100%;
+  border: none;
+  outline: none;
+  font-size: 16px;
+  font-family: "Avenir-Heavy", sans-serif;
```

Move `font-family` into html selector. For input use inherit value
```suggestion
  font-family: inherit;
```